gateway/1.6/modules/ROOT/pages/flex-troubleshoot-soap-apis.adoc

```diff
@@ -0,0 +1,29 @@
+= Publishing SOAP APIs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway does not natively support SOAP APIs and does not provide any schema validation for XML. However, you can publish an HTTP API instance to a secure SOAP upstream service.
```

Comment:
"...does not provide any schema validation for XML."

At first glance I assumed this was talking about the schema validation policy. Is it actually referring to validating SOAP XML? If so, I wonder if there's a way to make it a little clearer? Or, I could be misunderstanding.

Reply:
So, I brought this up with Federico and he felt it was clear enough, and I kind of agree.

From my understanding:

Mule Gateway auto-validates requests based on what the API type is.

Flex does not have any auto-validation. The only validation it provides is by using the schema validation policy. Which we mention isn't supported as well.